src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -447,6 +469,65 @@ describe('non-empty successful affected clusters table', () => {
       .eq(0)
       .should('have.text', 'dd2ef343-9131-46f5-8962-290fdfdf2199');
   });
+
+  // TODO fix test: double sorting? keep ordering in single sorting?
+  Object.entries({
+    name: 'Name',
```

Comment:
probably could be brought to a constant/variable

Reply:
Good idea, thank you. I propose to use 

```_.zipObject(['name', 'last_checked_at'], TABLE_HEADERS)```

Do you think it will make sense to define `['name', 'last_checked_at']` as a constant? It is only used in this test

Reply:
I see you use `Object.entries` on `zipObject`. Can it be replaced just with `zip` instead? The results will be the same: an array of 2-items-length arrays.

Reply:
If only in this test, then yeah, it doesn't make sense :-)

Reply:
Great suggestion, way better. Thank you!